portal/src/graphql/adminapi/AddPhoneScreen.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useCallback, useContext, useMemo, useState } from "react";
+import { useParams } from "react-router-dom";
+import { Dropdown, PrimaryButton, TextField, Toggle } from "@fluentui/react";
+import deepEqual from "deep-equal";
+import { Context, FormattedMessage } from "@oursky/react-messageformat";
+
+import NavBreadcrumb from "../../NavBreadcrumb";
+import NavigationBlockerDialog from "../../NavigationBlockerDialog";
+import ShowLoading from "../../ShowLoading";
+import ShowError from "../../ShowError";
+import UserDetailCommandBar from "./UserDetailCommandBar";
+import { useDropdown, useTextField } from "../../hook/useInput";
+import { useAppConfigQuery } from "../portal/query/appConfigQuery";
+import { PortalAPIAppConfig } from "../../types";
+
+import styles from "./AddPhoneScreen.module.scss";
+
+interface AddPhoneFormProps {
+  appConfig: PortalAPIAppConfig | null;
+}
+
+// fallback if config does not exist
+const DEFAULT_COUNTRY_CODE = "852";
```

Comment:
The default country calling code should not be 852, that kinda arbitrary and biased for international users. Let's follow the default value of config and use first country calling code in lexicographical order: `+1`.